peps/api/index.rst

```diff
@@ -1,17 +1,18 @@
 PEPs API
 ========
 
-There is a read-only API of published PEPs available at:
+There is a read-only JSON document of every published PEP available at
+https://peps.python.org/api/peps.json or https://peps.python.org/peps.json
+(both URLs point to the same document).
```

Comment:
Ah, this is a bug, it should only be at https://peps.python.org/api/peps.json and not https://peps.python.org/peps.json

Re: https://github.com/python/peps/issues/2584#issuecomment-1758957285

```suggestion
There is a read-only JSON document of every published PEP available at:

* https://peps.python.org/api/peps.json
```

Let's remove the one at the root. A GitHub code search only shows results for the correct https://peps.python.org/api/peps.json.

Reply:
I suggest moving this bit to its own PR (which I'll oppose ;) and keep this PR focused on the new `authors_list` (which I'll approve)?

Reply:
Done, to un-block this PR